data.sql

```diff
@@ -8,3 +8,15 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Pikachu', '01/07/2021', 15.04,false, 1 );
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Devimon', '05/12/2017', 11, true, 5);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Charmander', '08/02/2020', 11, false, 0);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Plantmon', '15/11/2021', 5.7, true, 2);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Squirtle', '04/02/1993', 12.13, false, 3);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Angemon', '06/12/2005', 45, true, 1);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Boarmon', '06/07/2005', 20.4, true, 3);
+INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
+VALUES ('Blossom', '14/05/2022', 22, true, 4);
```

Comment:
- [ ] Firstly, the date of birth for this animal `(Blossom)` should be `13/10/1998`.
- [ ] Secondly, the weight for this animal `(Blossom)` should be a `17.0` I am suggesting you update it.
- [ ] Finally, the escaped_attempts should be value `3` instead of `4`